o3de/o3de
# Multiplayer "Connect" Console Command Working Via Commandline and SetReg

Support calling the multiplayer 'connect' command via command-line and setreg by ensuring the command is registered once the Multiplayer System is ready. Fixes #14474

Depreciate 'cl_connect_onstartup' now that 'connect' works
See #14476 for deprecation notice

## How was this PR tested?
1. Open a server and run sv_lauch_local_client
2. Add _connect_ to settings.multiplayersample_gamelauncher.setreg and see that game automatically connects to a server
```
{
    "Amazon": {
        "AzCore": {
            "Runtime": {
                "ConsoleCommands": {
                    "connect": true
                }
            }
        }
    }
}

```
